google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -471,9 +481,10 @@ static DeleteRule fromPb(Rule rule) {
    * Delete rule class that sets a Time To Live for blobs in the bucket.
    *
    * @see <a href="https://cloud.google.com/storage/docs/lifecycle">Object Lifecycle Management</a>
-   *
-   * @deprecated Use a {@code LifecycleRule} with a {@code DeleteLifecycleAction} and use
-   * {@code LifecycleCondition.Builder.setAge} instead
+   * @deprecated Use a {@code LifecycleRule} with a {@code DeleteLifecycleAction} and use {@code
```

Comment:
Could you use `<p>` to space the javadoc a bit.

Example:
https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java#L704

Render:
https://googleapis.github.io/google-cloud-java/google-cloud-clients/apidocs/com/google/cloud/storage/BucketInfo.html#versioningEnabled--